### PR TITLE
LB-69: Add more installation instructions

### PR DESCRIPTION
**Pull request by alastair**
http://tickets.musicbrainz.org/browse/LB-69
This includes info about listenstore and messybrainz, although
many of these steps should be performed automatically on provision

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
...
+ListenStore is our interface to cassandra. It's bundled as part of this
+repository, but you must install it so that python can find it.
+
+    $ cd listenstore
+    $ python setup.py develop
+
+You can use `setup.py develop` to symlink the listenstore directory into
+site-packages (letting you make changes easily), or use `setup.py install`
+to install it. If you make any changes to listenstore you must install
+again.
```
Comment (mayhem) [inline README.md]: If you use "install" and make any changes to listenstore you must run install again.

File: README.md [hunk 2]
```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
...
+
+MessyBrainz is an intermediate database to map unknown metadata to stable
+UUIDs, and then map these intermediate identifiers to MBIDs
+(http://messybrainz.org, https://github.com/metabrainz/messybrainz-server)
```
Comment (mayhem) [inline README.md]: Spaces inside the parens to prevent dumbass browsers from doing the wrong thing

File: README.md [hunk 3]
```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
...
+
+For now you must download and install MessyBrainz separately:
+
+    $ git clone https://github.com/metabrainz/messybrainz-server.git
+    $ cd messybrainz-server
+    $ pip install -r requirements.txt
+    $ python setup.py develop
+
```
Comment (mayhem) [inline README.md]: add: python manage.py init_db

File: README.md [hunk 4]
```diff
@@ -14,33 +14,67 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
...
 
     $ python manage.py init_db
 
 After that server should be ready to go.
 
+Initialize the MessyBrainz database by changing to the messybrainz-server
+directory, copying config.py.sample to config.py, and running the same:
+
+    $ python manage.py init_db
+
```
Comment (mayhem) [inline README.md]: Ah, I see you have it here. It feels out of place here in a separate section where I have to go back to deal with messybrainz.

**Comment by alastair** [comment]
changed

File: README.md (hunk 2 above)
Comment (gentlecat) [inline README.md]: I don't think this is necessary. It looks just fine here: https://github.com/metabrainz/listenbrainz-server/blob/c9aad/README.md#messybrainz-and-listenstore. GitHub is adding link tags implicitly.

Comment (gentlecat) [inline README.md]: Even if that didn't work, it's better to just add links explicitly yourself.

File: README.md [hunk 5]
```diff
@@ -14,27 +14,59 @@ First, install [ChefDK](https://downloads.chef.io/chef-dk/), then:
...
+    $ python setup.py develop
+
+You can use `setup.py develop` to symlink the ListenStore directory into
+site-packages (letting you make changes easily), or use `setup.py install`
+to install it.  If you use `setup.py install` and make any changes to
+ListenStore you must run install again.
+
+MessyBrainz is an intermediate database to map unknown metadata to stable
+UUIDs, and then map these intermediate identifiers to MBIDs
+( http://messybrainz.org, https://github.com/metabrainz/messybrainz-server )
```
Comment (gentlecat) [inline README.md]: Period at the end of this sentence.

**Pull request merged**